core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.html

```diff
@@ -1,63 +1,96 @@
-                        icon="pi pi-ellipsis-v"
-                        attr.data-testid="bulkActions"
-                    ></button>
-
-                    <p-menu
-                        #actionsMenu
-                        [popup]="true"
-                        [model]="vm.containerBulkActions"
-                        appendTo="body"
-                    ></p-menu>
-                </div>
+<dot-portlet-base *ngIf="vm$ | async as vm">
+    <dot-action-header [options]="vm.actionHeaderOptions">
+        <dot-content-type-selector
+            (selected)="changeContentTypeSelector($event)"></dot-content-type-selector>
+        <input
+            #gf
+            (input)="handleQueryFilter($event.target.value)"
+            (keydown.arrowdown)="focusFirstRow()"
+            pInputText
+            placeholder="{{ 'global-search' | dm }}"
+            type="text"
+            data-testId="query-input" />
+        <div class="container-listing__header-options">
+            <div class="container-listing__filter-controls">
+                <p-checkbox
+                    [label]="'Show-Archived' | dm"
+                    [binary]="true"
+                    (onChange)="handleArchivedFilter($event.checked)"
+                    data-testId="archiveCheckbox"></p-checkbox>
             </div>
 
-            <ng-template #beforeSearchTemplate>
-                <dot-content-type-selector
-                    (selected)="changeContentTypeSelector($event)"
-                ></dot-content-type-selector>
-            </ng-template>
+            <div>
+                <button
+                    class="p-button-outlined p-button"
+                    [label]="'Actions' | dm"
+                    [disabled]="!selectedContainers.length"
+                    (click)="handleActionMenuOpen($event)"
+                    type="button"
+                    pButton
+                    icon="pi pi-ellipsis-v"
+                    data-testId="bulkActions"></button>
 
-            <ng-template #rowTemplate *ngIf="vm.tableColumns as tableColumns" let-rowData="rowData">
-                <td [ngStyle]="{ 'text-align': tableColumns[0].textAlign }">
+                <p-menu
+                    #actionsMenu
+                    [popup]="true"
+                    [model]="vm.containerBulkActions"
+                    appendTo="body"></p-menu>
+            </div>
+        </div>
+    </dot-action-header>
+    <p-table
+        *ngIf="vm.tableColumns && vm.actionHeaderOptions"
+        [(selection)]="selectedContainers"
+        [columns]="vm.tableColumns"
+        [value]="vm.containers"
+        [rows]="40"
+        [paginator]="true"
+        [pageLinks]="vm.maxPageLinks"
+        [totalRecords]="vm.totalRecords"
+        [responsiveLayout]="'scroll'"
+        (onPage)="loadDataPaginationEvent($event)"
+        selectionMode="multiple"
+        loadingIcon="pi pi-spin pi-spinner"
+        dataKey="inode"
+        data-testId="container-list-table">
```

Comment:
Why did you remove the `dot-listing-data-table`? That component already have the `dot-action-header`

Reply:
The flow we had before involved two different data sources, the table itself and the store, for this reason I use PrimeNG's native table, that way I can keep the data and its manipulation in the store

Reply:
The `dot-listing-data-table` should be a `presentational component` to display the data we passed to it. However, that can be discussed in another `issue`.